ui/pages/send/send-content/send-content.component.js

```diff
@@ -2,7 +2,8 @@ import React, { Component } from 'react';
 import PropTypes from 'prop-types';
 import PageContainerContent from '../../../components/ui/page-container/page-container-content.component';
 import Dialog from '../../../components/ui/dialog';
-import ActionableMessage from '../../../components/ui/actionable-message';
+import { BannerAlert } from '../../../components/component-library';
```

Comment:
Need to import `ButtonLink` as well
```suggestion
import { BannerAlert, ButtonLink } from '../../../components/component-library';
```

Reply:
Sorry, I overlooked earlier and committed your suggestions directly.